wcfsetup/install/files/lib/system/html/input/node/HtmlInputNodeTextParser.class.php

```diff
@@ -422,24 +422,7 @@ protected function parseURL(\DOMText $text, $value, $allowURL, $allowMedia)
     {
         static $urlPattern = '';
         if ($urlPattern === '') {
```

Comment:
This can be removed now and the regex inlined in the `preg_replace_callback` call, as the regex is a static string. The `$illegalChars` property can likely be deprecated.

Reply:
`$illegalChars` is still being used in `parseEmail()`